templates/samples/statusBar/index.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Generated using theia-plugin-generator
+ */ 
+import * as theia from '@theia/plugin';
+
+const disposables: theia.Disposable[] = [];
+
+export function start() {
+    registerCommand({
+        id: 'status_bar-message-command-timeout',
```

Comment:
we've to have something for "patterns"

we have quick-pick-something while here it's an underscore for status_bar